README.md

```diff
@@ -90,6 +90,7 @@ This action requires the `pull-requests: write` permission to add a comment to y
 | `name`                      | Give the report a custom name. This is useful if you want multiple reports for different test suites within the same PR. Needs to be unique.                                                                                                                     | ''                                                                                                                                                                                                                                                                 |
 | `pr-number`                 | The number of the PR to post a comment to. When using the `push` trigger, you can set this option to "auto" to make the action automatically search of a PR with a matching `sha` value and comment on it.                                                                                                                                                                                                              | If in the context of a PR, the number of that PR.<br/> If in the context of a triggered workflow, the PR of the triggering workflow.                                                                    <br/>If no PR context is found, it defaults to `undefined` |
 | `comment-on`                | Specify where you want a comment to appear: "pr" for pull-request (if one can be found), "commit" for the commit in which context the action was run, or "none" for no comments. You can provide a comma-separated list of "pr" and "commit" to comment on both. | `pr`                                                                                                                                                                                                                                                               |
+| `threshold-icons`           | A JSON object mapping coverage percentage thresholds to status icons. When no vitest coverage threshold is defined, this determines the icon shown. See [Threshold Icons](#threshold-icons) for details.                                                          | ''                                                                                                                                                                                                                                                                 |
```

Comment:
This documentation is still missaligned with implmentation. Threshold icons will always trake precendence over the threshold - please state this here accordingly.